lib/discord2telegram/setup.js

```diff
@@ -68,16 +68,17 @@ function setup(dcBot, tgBot, dcUsers, messageMap) {
 		if (senderId !== dcBot.user.id) {
 
 			// Check if the message is from the correct chat
-			if (message.channel.id === Application.settings.discord.channelID) {
+			if (Application.chats.has("discord", message.channel.id)) {
+				const chat = Application.chats.getChat("discord", message.channel.id);
 
 				// Check for attachments and pass them on
 				message.attachments.forEach(({url}) => {
 					tgBot.sendMessage({
-						chat_id: Application.settings.telegram.chatID,
+						chat_id: Number(chat.telegram),
```

Comment:
Maybe do `chat.telegram = Number(chat.telegram)` once in the init of the map, instead of doing it every time it is needed?